File: LeetCode/Python/#633 Sum of Square Numbers.py
import math
class Solution:
    def judgeSquareSum(self, c):
        """
        :type c: int
        :rtype: bool
        """
        mid =math.floor(math.sqrt(c/2))+1;
        for i in range(mid):
            tmp = math.sqrt(c - pow(i,2))
            if tmp == int(tmp):
                return True
        return False
                
            

File: LeetCode/JavaScript/#633 Sum of Square Numbers.js
/**
 * @param {number} c
 * @return {boolean}
 */

var judgeSquareSum = function(c) {
	var mid = ~~(Math.sqrt(c/2));
    for(var i = 0; i <= mid;i++){
    	if(Number.isInteger(Math.sqrt(c-Math.pow(i,2))))
    		return true;
    }
    return false;
};